src/Neo/ProtocolSettings.cs

```diff
@@ -70,7 +70,7 @@ public record ProtocolSettings
         /// <summary>
         /// The maximum increment of the <see cref="Transaction.ValidUntilBlock"/> field.
         /// </summary>
-        public uint MaxValidUntilBlockIncrement => 86400000 / MillisecondsPerBlock;
+        public uint MaxValidUntilBlockIncrement => 86400000 / MillisecondsPerBlock; //TODO keep the same??
```

Comment:
Can be different after the HF? (Or related to MsPerBlock)

Reply:
It's safe to make it bigger, won't invalidate any old transactions. The intent was to have a day of blocks. In most scenarios it's an overkill (I specifically prefer short transaction lifetimes, 10 blocks is more than enough), but in some cases it's still relevant, consider manually (hi, #1573) signing a committee transaction.

One caveat is that it can not be more than `MaxTraceableBlocks`, otherwise replays would be possible.

Reply:
But now how many years we will preserve?

Reply:
`MaxTraceableBlocks` is 2102400 irrespective of block time, but I really (_really_) would like it to be lower, as in #3644. With MVUBI of 28800 (as per 3s MillisecondsPerBlock) we can have MTB of 30K. Or we can lower MVUBI to 14400 and have MTB of 20K. Either way it's all per-network, FS networks already use pretty low MTB (and testnet runs at 1s block time) and we love it.

Reply:
Notice also that it's all configurable, we don't need to change anything in the code, just update settings.